recipe/build.sh

```diff
@@ -8,10 +8,8 @@ do
     -DCMAKE_PREFIX_PATH=${PREFIX} \
     -DCMAKE_INSTALL_PREFIX=${PREFIX} \
     -DBUILD_SHARED_LIBS=${shared_libs} \
-    -DCMAKE_BUILD_WITH_INSTALL_RPATH=ON \
     ..
 
-  make -j${CPU_COUNT}
+  make install -j${CPU_COUNT}
 done
-cp lib/libarpack${SHLIB_EXT} libarpack.a ${PREFIX}/lib
 DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib make check -j${CPU_COUNT}
```

Comment:
`DYLD_FALLBACK_LIBRARY_PATH` is getting phased out as it doesn't work on newer macOS versions because of SIP. Instead would add `-Wl,-rpath,$PREFIX/lib` to `LDFLAGS` on macOS. Linux would be `-Wl,-rpath=$PREFIX/lib`.